plugins/http_plugin/http_plugin.cpp

```diff
@@ -304,18 +304,20 @@ namespace eosio {
                               [ioc = this->server_ioc, &bytes_in_flight = this->bytes_in_flight, handler_itr,
                                resource{std::move( resource )}, body{std::move( body )}, con]() {
                      try {
-                        bytes_in_flight -= body.size();
                         handler_itr->second( resource, body,
-                              [ioc{std::move(ioc)}, &bytes_in_flight, con]( int code, std::string response_body ) {
-                           bytes_in_flight += response_body.size();
-                           boost::asio::post( *ioc, [ioc, response_body{std::move( response_body )}, &bytes_in_flight, con, code]() {
-                              size_t body_size = response_body.size();
-                              con->set_body( std::move( response_body ) );
+                              [ioc{std::move(ioc)}, &bytes_in_flight, con]( int code, fc::variant response_body ) {
+                           boost::asio::post( *ioc, [ioc, response_body{std::move( response_body )}, &bytes_in_flight, con, code]() mutable {
+                              std::string json = fc::json::to_string( response_body );
```

Comment:
Can this be const? (or does move prevent that?)

Reply:
The move prevents it.